tests/scripts/vector-from-tasty

```diff
@@ -0,0 +1,5 @@
+mkdir -p "$PROG_HOME/tests/vector-out"
+
+compile -d "$PROG_HOME/tests/vector-out" "$PROG_HOME/tests/Vector.scala"
+
+measure "from-tasty/Vector.scala" scala.collection.immutable.Vector scala.collection.immutable.VectorIterator scala.collection.immutable.VectorBuilder scala.collection.immutable.VectorPointer -classpath "$PROG_HOME/tests/vector-out" -from-tasty
```

Comment:
There is an exception while jhm is running:

```
# Warmup Iteration   1: not found: scala.collection.immutable.Vector
not found: scala.collection.immutable.VectorIterator
not found: scala.collection.immutable.VectorBuilder
not found: scala.collection.immutable.VectorPointer
```

I checked the folder, it seems the class files are there:

```
fliu@lamppc37:/data/workspace/bench$ ls tests/vector-out/scala/collection/immutable/Vector
Vector$$anon$1.class     VectorBuilder.hasTasty   Vector$.class            VectorIterator.class     VectorPointer.class
VectorBuilder.class      Vector.class             Vector.hasTasty          VectorIterator.hasTasty  VectorPointer.hasTasty
```

Any ideas @nicolasstucki ?